docs/rst/knora-api-server/design-documentation/administration.rst

```diff
@@ -769,23 +769,110 @@ All information regarding users, projects and groups is stored in the ``http://w
 
 
 Users Endpoint
-^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^
+
+**Get users**
+  - GET: ``/v1/users``
+
+**Get user**
+  - GET:``/v1/users/<userIri>``
+
 **Create user**:
   - Required permission: none, self-registration is allowed
-  - Required information: username, given name, family name, email, password
-  - Optional information: phone
+  - Required information: email, given name, family name, password
   - Returns IRI of newly created user
+  - POST: ``/v1/users/``
+  - BODY:
+::
 
-
-**Update user**:
+  {
+    "email": "donald.duck@example.org",
+    "givenName": "Donald",
+    "familyName": "Duck",
+    "password": "test",
+    "status": true,
+    "lang": "en",
+    "systemAdmin": false
+  }
+
+**Update user's basic information**:
   - Required permission: SystemAdmin / User
-  - Changeable information: username, given name, family name, email, password, phone
+  - Changeable information: email, given name, family name
+  - PUT: ``/v1/users/userIri``
+  - BODY:
+::
+
```

Comment:
Instead of writing examples by hand, you could refer to the typescript interfaces. But you can also do both (maybe a concrete example is useful).